src/server.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const express = require('express');
+const { userRouter } = require('./routes/users');
+const { expenseRouter } = require('./routes/expenses');
+
+const server = express();
+
+server.use('/users', userRouter);
+server.use('/expenses', expenseRouter);
+
+server.listen(3000);
```

Comment:
if you'll decide to create .env file you could also add env variable for PORT and pass it here